diffsims/generators/rotation_list_generators.py

```diff
@@ -58,6 +58,44 @@ def get_fundemental_zone_grid(space_group_number, resolution):
     fz_grid_axangle = reduce_to_fundemental_zone(raw_grid_axangle, zone_string)
     return _returnable_eulers_from_axangle(fz_grid_axangle,'rzxz',round_to=2)
 
+def get_grid_streographic(crystal_system,resolution,equal='angle'):
+    """
+    Creates a rotation list by combining the minimum region of the streogram's beam directions
+    with in plane rotations
```

Comment:
I think this would be fine if we dropped "and for a specified crystal symmetry" at the end. We don't use the symmetry of the crystal for the inplane rotations.